app/views/agent/partials/Covid.scala.html

```diff
@@ -18,21 +18,28 @@
 
 @this()
 
-@()(implicit messages: Messages, appConfig: AppConfig)
+@(preCovidDeadline: Boolean)(implicit messages: Messages, appConfig: AppConfig)
 
 <div id="covid-partial" class="warning-banner">
   <div class="notice">
     <i class="icon icon-important">
       <span class="visually-hidden">@messages("common.warning")</span>
     </i>
-    <p><strong class="bold-small">@messages("covid.title")</strong></p>
+      @if(preCovidDeadline) {
```

Comment:
The logic is correct here but slightly confusing naming - if `preCovidDeadline` then the `postEnd` message is returned?